{
	"about": "https://jvndb.jvn.jp/ja/contents/2021/JVNDB-2021-011145.html",
	"title": "Zoho ManageEngine ADSelfService Plus におけるデータの信頼性についての不十分な検証に関する脆弱性",
	"link": "https://jvndb.jvn.jp/ja/contents/2021/JVNDB-2021-011145.html",
	"description": "Zoho ManageEngine ADSelfService Plus には、データの信頼性についての不十分な検証に関する脆弱性が存在します。",
	"identifier": "JVNDB-2021-011145",
	"references": [
		{
			"text": "https://www.cve.org/CVERecord?id=CVE-2021-37421",
			"source": "CVE",
			"id": "CVE-2021-37421"
		},
		{
			"text": "https://nvd.nist.gov/vuln/detail/CVE-2021-37421",
			"source": "NVD",
			"id": "CVE-2021-37421"
		},
		{
			"text": "https://blog.stmcyber.com/vulns/cve-2021-37421/",
			"source": "関連文書",
			"id": "CVE-2021-37421"
		},
		{
			"text": "https://cwe.mitre.org/data/definitions/345.html",
			"id": "CWE-345",
			"title": "データの信頼性についての不十分な検証(CWE-345)"
		}
	],
	"cpe": [
		{
			"text": "cpe:/a:zohocorp:manageengine_adselfservice_plus",
			"version": "2.2",
			"vendor": "Zoho Corporation",
			"product": "ManageEngine ADSelfService Plus"
		}
	],
	"cvss": [
		{
			"version": "2.0",
			"score": "7.5",
			"type": "Base",
			"severity": "High",
			"vector": "AV:N/AC:L/Au:N/C:P/I:P/A:P"
		},
		{
			"version": "3.0",
			"score": "9.8",
			"type": "Base",
			"severity": "Critical",
			"vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
		}
	],
	"date": "2022-07-19T15:47+09:00",
	"issued": "2022-07-19T15:47+09:00",
	"modified": "2022-07-19T15:47+09:00"
}
